### ISSUE TITLE
No way to translate message timeout-or-duplicate

### ISSUE THREAD

**Issue by ebelendez** [RyanDaDeng/laravel-google-recaptcha-v3#65]
If I let 2 minutes pass before I submit the form, the response comes with this message:
timeout-or-duplicate

I suppose it is normal but I don't find how to translate it, Am I missing something?

My temporary solution was to modify the GoogleReCaptchaV3ValidationRule / passes(), like this:

```
public function passes($attribute, $value)
    {
        $response = GoogleReCaptchaV3::setAction($this->action)->verifyResponse($value, app('request')->getClientIp());
        $this->message = $response->getMessage();
        if($this->message == 'timeout-or-duplicate')$this->message = trans('GoogleReCaptchaV3::messages.TIMEOUT_OR_DUPLICATE');
        return $response->isSuccess();
    }
```
Is this a bug? or some google update? What do you suggest?

**Comment by RyanDaDeng** [comment]
Thanks! @ebelendez I will give you an update by today or tmr.

**Comment by RyanDaDeng** [comment]
Fixed, please install 2.3.1

The trans that I defined in lang is for api connection timeout errors.

I have added `timeout-or-duplicate` error trans. The default message is `The action is timeout.`

**Comment by ebelendez** [comment]
Cool thanks!
